test/test_testmon.py

```diff
@@ -25,7 +23,7 @@ def test_separation(self, testdir):
         testmon2_data.write_data()
 
         testmon_check_data = CoreTestmonData(testdir.tmpdir.strpath, variant='1')
-        assert testmon1_data.node_data['node1'] == {'a.py': 1}
```

Comment:
?

Reply:
`testmon_check_data` is unused here - should just the assignment get skipped?
(I've assumed that the `assert` should be for `testmon_check_data`)

Reply:
@tarpas 
Please verify that this is sane, and I will rebase this.

Reply:
I guess this test should check if data is not overwritten when different variant is used. But the data is never written to DB since condition in `write_data()` in not met. Also `testmon_check_data.read_data()` is never called.